standalone/src/test/java/io/pravega/local/InProcPravegaClusterTest.java

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.local;
+
+import io.pravega.client.admin.StreamManager;
+import io.pravega.client.stream.ScalingPolicy;
+import io.pravega.client.stream.StreamConfiguration;
+import io.pravega.segmentstore.server.store.ServiceBuilderConfig;
+import io.pravega.test.common.TestUtils;
+import java.net.URI;
+import lombok.Cleanup;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * Unit tests for standalone
+ */
+@Slf4j
+public class InProcPravegaClusterTest {
+    LocalPravegaEmulator localPravega;
+
+    @Before
+    public void setUp() throws Exception {
+        ServiceBuilderConfig config = ServiceBuilderConfig
+                .builder()
+                .include(System.getProperties())
```

Comment:
This is a unit test to which we shouldn't pass any sys props. Why is this necessary?

Reply:
This is to ensure that the test is as near to `standalone` as possible.